roles/openshift_facts/vars/main.yml

```diff
@@ -2,6 +2,7 @@
 required_packages:
   - iproute
   - python-dbus
+  - python-six
```

Comment:
@rhcarvalho either way works for me, I was mainly trying to address the immediate issue with this PR.